### DISCUSSION TITLE
Backup question

### DISCUSSION THREAD

**Discussion by eightwt** [jendrikseipp/rednotebook#618] [Ideas]
Is there any simple way to customize the backup function so that I can change the interval? I'd like to be able to backup maybe every 15th day or 20th day instead of every 8th.

**Comment by jendrikseipp** [comment]
Not within the app, no. But with an external command, e.g., a cron job, you can schedule backups at any interval. Backing up the journal is simply done by zipping the journal directory under ~/.rednotebook/data (default) and moving it to your desired backup location.
